scripts/ci/libraries/_build_images.sh

```diff
@@ -800,6 +624,12 @@ function build_images::build_prod_images() {
         echo
         exit 1
     fi
+    if [[ ${PREPARE_BUILDX_CACHE} == "true" ]]; then
+        # Cache for prod image contains also build stage for buildx when mode=max specified!
+        docker_cache_prod_directive+=(
+            "--cache-to=type=registry,ref=${AIRFLOW_PROD_IMAGE}:cache,mode=max"
```

Comment:
Do you consider to use `gha` cache?
> gha type type exports cache through [the Github Actions Cache service API](https://github.com/tonistiigi/go-actions-cache/blob/master/api.md#authentication).

https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#-export-build-cache-to-an-external-cache-destination---cache-to

Reply:
Yep. I did consider it, but:

> Github Actions cache saves both cache metadata and layers to GitHub's Cache service. This cache currently has a size limit of 10GB that is shared accross different caches in the repo. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 10 GB. Recycling caches too often can result in slower runtimes overall.